tests/statements/convert.rs

```diff
@@ -0,0 +1,75 @@
+use crate::statements::*;
+
+#[tokio::test]
+async fn test_convert_from_flat_parquet_table() -> Result<(), DataFusionError> {
+    let (context, maybe_test_dir) = make_context_with_pg(ObjectStoreType::Local).await;
+
+    // Prepare a flat Parquet table
+    let table_uuid = Uuid::new_v4();
+    let temp_dir = maybe_test_dir.expect("temporary data dir exists");
+    let table_path = temp_dir.path().join(table_uuid.to_string());
+    // Create the directory as otherwise the COPY will fail
+    create_dir(table_path.clone()).await?;
+
+    // COPY some values multiple times to test converting flat table with more than one parquet file
+    context
+        .plan_query(&format!(
+            "COPY (VALUES (1, 'one'), (2, 'two')) TO '{}/file_1.parquet'",
```

Comment:
I didn't realize we / DataFusion could do that 😅

Reply:
Yup, ever since https://github.com/splitgraph/seafowl/pull/462